src/Editor/CodelensProvider.ts

```diff
@@ -43,6 +44,7 @@ export class CodelensProvider implements vscode.CodeLensProvider {
       let regex = new RegExp(/(.+)/g);
       let activatedEditorText = document.getText();
       let matches;
+      let nowToolName = '';
```

Comment:
is the meaning `now` be current  or active or selected tool?
how about using them instead of `now` ?

Reply:
I was also worried about the variable name...
It is used to tell which tool the attribute comes from.
```
[one-optimize]
input_path=
output_path=

[one-codegen]
backend=
command=
```
When the code is the same as above, `nowToolName` is updated in `[one-optimize]`. 
And we use this in `input_path` and `output_path`.
It's also the same in `one-codegen`.
Therefore, it is defined as `nowToolName` because it is the name of the current tool to process the value for the attribute.

Should I just define it as `toolName`?

Reply:
>  current tool to process

As you wrote, `currentToolName` seems nice.